payjoin/src/core/receive/mod.rs

```diff
@@ -423,7 +423,7 @@ impl OriginalPayload {
     }
 
     pub fn identify_receiver_outputs(
-        &self,
+        self,
```

Comment:
Why is this self changed to no longer be a ref in this commit as opposed to in b8b2a42?